src/AltinnCore/Common/Services/Implementation/ExecutionAppSI.cs

```diff
@@ -41,7 +41,7 @@ public class ExecutionAppSI : IExecution
         /// <summary>
         /// Initializes a new instance of the <see cref="ExecutionAppSI"/> class.
         /// </summary>
-        /// <param name="settings">The repository setting service needed (set in startup.cs)</param>
+        /// <param name="settings">The ServiceRepositorySettings needed (set in startup.cs)</param>
```

Comment:
i would suggest to change it back to the way it was. Use descriptive sentences rather than class names.

Reply:
"The app repository settings."